### PR TITLE
Use SQLite instead of Postgres

### PR DESCRIPTION
**Pull request by EmilioCristalli**
This is a proposal to start using SQLite instead of Postgres, to simplify the setup and deployment a little bit.

Since this is just a demo app, it feels like it doesn't really need a Postgres server, and SQLite might just be enough.

SQLite can probably be tweaked to have better performance, but felt like it wasn't worth it for this case.

For the Kamal/Docker container i decided not to map the db file to the host machine (so the DB will be lost on every deploy) to keep things simple, but also thinking it might not be a bad idea to reset the DB now and then (open to discussion though!)

### CODE REVIEW

File: Dockerfile
Comment (santiagorodriguez96) [inline Dockerfile:1]: Shouldn't we update the Dockerfile to install `sqlite3` package and remove the `libpq-dev` package? I managed to successfully build the image tho without those changes

**Review by santiagorodriguez96 — COMMENTED** [review]
LGTM

File: Dockerfile
Comment (santiagorodriguez96) [reply Dockerfile:1]: Ahh I see that now the `sqlite3` gem comes with the library packaged – nice!

So we should just remove the `libpq-dev` package 🙂

**Review by santiagorodriguez96 — APPROVED** [review]
LGTM

File: Dockerfile
Comment (EmilioCristalli) [reply Dockerfile:1]: 👍  removed `libpq-dev` in 6e3d38c

And I think sqlite doesn't need any pre installed OS packages. I think the gem has everything it needs to work with SQLite

**Pull request merged**